cmd/tools/commentMonitor/main.go

```diff
@@ -66,27 +66,25 @@ func main() {
 		args = append(args, strconv.Itoa(prnumber))
 		writeArgs(args, *output)
 
-		// Posting benchmark start comment.
-		comment := fmt.Sprintf(`Welcome to Prometheus Benchmarking Tool.
-
-The two prometheus versions that will be compared are _**pr-%v**_ and _**%v**_
-
-The logs can be viewed at the links provided in the GitHub check blocks at the end of this conversation
-
-After successfull deployment, the benchmarking metrics can be viewed at :
-- [prometheus-meta](%v/prometheus-meta) - label **{namespace="prombench-%v"}**
-- [grafana](%v/grafana) - template-variable **"pr-number" : %v**
+		env := os.Environ()
+		envvar := make(map[string]string)
+		for _, e := range env {
+			tmp := strings.Split(e, "=")
+			envvar[tmp[0]] = tmp[1]
+		}
 
-The Prometheus servers being benchmarked can be viewed at :
-- PR - [prombench.prometheus.io/%v/prometheus-pr](%v/%v/prometheus-pr)
-- %v - [prombench.prometheus.io/%v/prometheus-release](%v/%v/prometheus-release)
+		unformatedComment := os.Getenv("TEMPLATE_BENCH")
```

Comment:
this needs to be passed as an arg as well.


```
template := app.Arg("template", "Template text so use when posting a github comment. All env variables will be replaces with.......").Required().String()

var unformatedComment string
unformatedComment = os.Getenv(template)
if unformatedComment ==""{
	unformatedComment = os.ReadFile(template)
}

if unformatedComment == "" ...fail
```

Reply:
Does it have to be an argument or is `flag` enough?

Reply:
yes flag is better.